draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -325,8 +332,12 @@ criteria in the remainder of this document might not apply.
 
 # Evaluation Criteria {#evaluation-criteria}
 
-As noted above, authors are expected to do a well-rounded evaluation of the pros
-and cons of congestion control algorithms that are brought to the IETF. The
+Authors are expected to evaluate the congestion control algorithms that
+they bring to the IETF.
+A well-rounded evaluation of the pros and cons of an algorithm is expected
+by a working group before it submits the specification for publication by the IETF.
+
```

Comment:
You made sure to use the active form in the previous sentence, and now we are back to passive. How about:

A working group is expected to perform a well-rounded evaluation of the pros and cons of an algorithm
before it submits the specification for publication by the IETF.